pkg/reconciler/trigger/controller.go

```diff
@@ -60,9 +59,6 @@ var filterBroker = pkgreconciler.AnnotationFilterFunc(eventingv1beta1.BrokerClas
 
 type envConfig struct {
 	ProjectID string `envconfig:"PROJECT_ID"`
-
-	MinRetryBackoff time.Duration `envconfig:"MIN_RETRY_BACKOFF" default:"1s"`
-	MaxRetryBackoff time.Duration `envconfig:"MAX_RETRY_BACKOFF" default:"1m"`
```

Comment:
We may want to keep these as defaults for brokers that don't have values for delivery spec fields, though they don't have to be env vars. They could be moved to constants.

Reply:
+1